docs/appendices/release-notes/5.10.8.rst

```diff
@@ -63,3 +63,6 @@ Fixes
 
 - Fixed an issue that caused queries to not complete and return if a 
   ``CircuitBreakerException`` was thrown while constructing the result set.
+
+- Fixed an issue that caused a bulk insert interrupted by ``KILL`` or
+  a node restart to not be cleared from the :ref:`sys.jobs <sys-jobs>` table.
```

Comment:
Isn't this also the case when some error happens during executing? And isn't this also related to update + delete?
```suggestion
- Fixed an issue that caused an interrupted bulk operation to not be cleared
  from the :ref:`sys.jobs <sys-jobs>` table.
```